lib/controllers/status-bar-tile-controller.js

```diff
@@ -88,6 +94,7 @@ export default class StatusBarTileController extends React.Component {
     return (
       <Fragment>
         {this.renderTiles(repoProps)}
+        <GithubStatusBarTile didClick={this.props.toggleGithubTab} />
```

Comment:
How about moving the GitHub tile to the far right?

![screen shot 2018-09-26 at 1 55 36 pm](https://user-images.githubusercontent.com/378023/46058259-23d13080-c195-11e8-84af-ffcb8176f13a.png)

Then it's more in order of this typical 🤔 workflow:
1. Choose a branch
2. Make sure it's up-to-date
3. Make changes
4. Create PR

Reply:
@annthurium That's a good point, although the "far right" sometimes gets used by the "updates" and "blue squirrel".

Another option would be to move "GitHub" to the left, before the branch name:

![image](https://user-images.githubusercontent.com/378023/46266344-66b25000-c569-11e8-95fd-016cbcde15e7.png)

Just so that "GitHub" doesn't get sandwiched between the Git "actions" and the Git tab.